Bump to version 3.2.0

awvwgk opened the pull request:
<!--
Thank you for pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* ~~Bumped the build number (if the version is unchanged)~~
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->
Closes https://github.com/conda-forge/dftd4-python-feedstock/pull/8
Closes https://github.com/conda-forge/dftd4-python-feedstock/pull/7
Closes https://github.com/conda-forge/dftd4-python-feedstock/issues/6

<!--
Please add any other relevant info below:
-->

awvwgk committed:
Bump to version 3.2.0

MNT: Re-rendered with conda-build 3.21.4, conda-smithy 3.10.1, and conda-forge-pinning 2021.05.01.06.26.24

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

awvwgk committed:
Don't require pkgconfig to setup the package

awvwgk committed:
Disable OSX

MNT: Re-rendered with conda-build 3.21.4, conda-smithy 3.10.1, and conda-forge-pinning 2021.05.01.06.26.24

awvwgk committed:
Enable testing for PyPy as well

awvwgk committed:
Reenable OSX

awvwgk committed:
Disable CPython 3.6 and 3.7 for OSX
